projects/go-style-guide/src/app/features/ui-kit/components/action-sheet-docs/components/action-sheet-overview/action-sheet-overview.component.html

```diff
@@ -135,7 +135,6 @@ <h2 class="go-heading-2">Placement</h2>
         <div class="go-column--100 go-column go-column--no-padding">
           <p class="go-body-copy">
             A simple example to demo different placements defined by <code class="code-block--inline">placement</code>.
-            For example, to define a right placement action sheet the code would be:
             <code [highlight]="placementGenericHtml"></code>
           </p>
```

Comment:
Do we want to add a `<h4 class="go-heading-4 go-heading--underlined">Code</h4>` here?

Reply:
Sure, I went ahead and added that in.